packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -187,49 +187,49 @@ export class CommonFrontendContribution implements MenuContribution, CommandCont
         });
     }
 
-    registerKeyBindings(registry: KeybindingRegistry): void {
+    registerDefaultKeyBindings(registry: KeybindingRegistry): void {
         if (supportCut) {
-            registry.registerKeyBinding({
+            registry.registerDefaultKeyBinding({
                 commandId: CommonCommands.CUT.id,
-                keyCode: KeyCode.createKeyCode({ first: Key.KEY_X, modifiers: [Modifier.M1] })
+                keyCode: TheiaKeyCodeUtils.createKeyCode({ first: Key.KEY_X, modifiers: [Modifier.M1] })
```

Comment:
let's just assign a string in our keybinding syntax here. The name should be changed from keyCode to e.g. `binding`. And get rid of `accelerator`, too.